apps/aebytecode/src/aebytecode.app.src

```diff
@@ -1,15 +0,0 @@
-{application, aebytecode,
- [{description, "Bytecode definitions for AEthernity VM shared with compiler."},
-  {vsn, "0.1.0"},
```

Comment:
I see now 1.0.0 - ok.

Reply:
The first running version, already in mainnet